stage1/init/common/kvm_mount.go

```diff
@@ -170,11 +171,21 @@ func AppToSystemdMountUnits(root string, appName types.ACName, volumes []types.V
 		whatFullPath := filepath.Join(root, whatPath)
 
 		if vol.Kind == "empty" {
-			diag.Printf("creating an empty volume folder for sharing: %q", whatFullPath)
-			err := os.MkdirAll(whatFullPath, 0700)
+			log.Printf("creating an empty volume folder for sharing: %q", whatFullPath)
```

Comment:
For the record, things printed with `log` always appear on the output, if `debug` is passed, a structured message will be printed.

Things printed with `diag` will only appear if debug is passed, which I thinks is what we want here.